tests/test_rw_functional.py

```diff
@@ -318,8 +318,11 @@ def test05ModifyStatus(run_cli, backends):
     assert bug.longdescs[-1]["text"] == comment
     assert bug.longdescs[-1]["is_private"] == 0
 
-    # Confirm comments is same as getcomments
+    # Confirm comments is same as get_comments
+    assert bug.comments == bug.get_comments()
+    # This method will be removed in a future version
     assert bug.comments == bug.getcomments()
+    assert bug.get_comments == bug.getcomments()
```

Comment:
I'm not sure whether I understand the purpose of this assertion: The method object on the left should be equal to the return value of the method call on the right?

I would suggest to compare the return values of both method calls. If you liked my other suggestion, you could even compare the two actual method objects.

Reply:
Thank you.